recipes/libctl/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+export CFLAGS="-I${PREFIX}/include ${CFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
```

Comment:
You shouldn't need to do this if you are using toolchain

Reply:
Fixed.